### DISCUSSION TITLE
Trusted Proxy setting won't survive reboot

### DISCUSSION THREAD

**Discussion by castigo86** [nextcloud/all-in-one#5076] [Questions] [answered]
### Steps to reproduce
1. Set up a working AIO instance behind a reverse proxy run an another server on local LAN. Check functionality.
2. All logins in reports are reported form the proxy IP. Bruteforce security will defend server from proxy address
3. Check default trusted proxies with `docker exec --user www-data -it nextcloud-aio-nextcloud php occ --private config:list` and it should report an array with IPs for localhost (127.0.0.1 etc etc).
4. Set up proxy address with command `docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:system:set trusted_proxies 0 --value='xxx.xxx.xxx.xxx/24'`
5. Confirm that value is set in the array.
6. Confirm IP is no longer throttled
7. Restart containers from AIO / Reboot server
8. run  `docker exec --user www-data -it nextcloud-aio-nextcloud php occ --private config:list`
9. confirm that values have returned to default as in point 3.

### Expected behavior 
New trusted proxy values set in point 4 should survive reboot.

### Actual behavior 
Trusted Proxy values in array are reset to default

### Host OS 
Debian 12 latest stable

#### Nextcloud AIO version 
9.3.0
#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2045

**Comment by castigo86** [comment]
Thank you!

**Comment by castigo86** [comment]
Despite the indication relating to the external IP not being relayed to nextcloud, my issue was about the trusted proxy setting not surviving a reboot.

The settings survive if set to a completely new string. I used the following command and it survived:

`sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:system:set trusted_proxies 20 --value="xxx.xxx.xxx.xxx/xx"`

Hope it helps
